src/app/backend/resource/logs/source.go

```diff
@@ -14,40 +14,37 @@
 
 package logs
 
-import "github.com/kubernetes/dashboard/src/app/backend/resource/generic"
+import "github.com/kubernetes/dashboard/src/app/backend/resource/controller"
 import "github.com/kubernetes/dashboard/src/app/backend/resource/common"
 import client "k8s.io/client-go/kubernetes"
 import meta "k8s.io/apimachinery/pkg/apis/meta/v1"
 import "k8s.io/client-go/pkg/api/v1"
 
-// Returns all log sources for a given resource. A log source identifies a log file through the combination of pod & container
-func GetLogSources(k8sClient *client.Clientset, ns string, resourceName string, resourceType string) (generic.LogSources, error) {
+// GetLogSources returns all log sources for a given resource. A log source identifies a log file through the combination of pod & container
+func GetLogSources(k8sClient *client.Clientset, ns string, resourceName string, resourceType string) (controller.LogSources, error) {
 	if resourceType == "Pod" {
 		return getLogSourcesFromPod(k8sClient, ns, resourceName)
-	} else {
-		return getLogSourcesFromController(k8sClient, ns, resourceName, resourceType)
 	}
+	return getLogSourcesFromController(k8sClient, ns, resourceName, resourceType)
 }
 
-// Returns all containers for a given pod
-func getLogSourcesFromPod(k8sClient *client.Clientset, ns, resourceName string) (generic.LogSources, error) {
-	logSources := generic.LogSources{}
+// GetLogSourcesFromPod returns all containers for a given pod
```

Comment:
Private methods do not need to start with method name. This is only required (by go linter) for public methods.